include/nighthawk/adaptive_load/nighthawk_service_client.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+#include "envoy/common/pure.h"
+
+#include "external/envoy/source/common/common/statusor.h"
+#include "external/envoy/source/common/protobuf/protobuf.h"
+
+#include "api/client/options.pb.h"
+#include "api/client/service.grpc.pb.h"
+
+namespace Nighthawk {
+
+/**
+ * An interface for interacting with a Nighthawk Service gRPC stub.
```

Comment:
Can you add a line here indicating whether or not this class is thread-safe?

Reply:
Added a comment to the impl.